### ISSUE TITLE
Data Science Consultant

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#3375]
Data Science Consultant<br>
http://ift.tt/2gXGl5I<br><p>The Data Science Consultant (DSC) works directly with clients and project teams, providing essential insight and leverage on problems where data science expertise is essential.&nbsp; S/he works to understand, describe and manage the lifecycle of analytics on a project from start to finish.&nbsp; While deeply involved with the technical solution, a DSC works equally on the execution, communication and delivery of a client project, often acting as the team&rsquo;s de facto tech lead.</p>
<br><p>As DSCs progress professionally, they tend towards greater depth in both data scientific principles and skills as well as domain-specific expertise, &lsquo;majoring&rsquo; in a subset of industries and becoming increasingly recognized as pivotal individuals in pushing forward the frontier of our capabilities to crack increasingly broad problems &ndash; and, in doing so, to shape the evolution of the firm and the industries it serves.</p>
<br><p>DSCs thus develop the expertise to know which quantitative problems are the most valuable in a certain industry, to understand which features and model structures are best suited to solving them, and to know how to tune such solutions in light of industry specific requirements and ecosystems.&nbsp; Ultimately, DSCs thus act as a crucial engine of progress, designing and maintaining assets and problem solving platforms.</p>
<br><p><strong>Skills &amp; Requirements</strong></p>
<br><ul>
<li><br></li>
<li>Technical background in computer science, data science, machine learning, artificial intelligence, statistics or other quantitative and computational science</li>
<li><br></li>
<li>Professional experience building and delivering data science and statistics based solutions and products, with an emphasis on creating measurable results and lasting impact<br><ul>
<li><br></li>
<li>Demonstrated ability to deliver technical projects with a team, often working under tight time constraints to deliver value</li>
<li><br></li>
<li>An &lsquo;engineering&rsquo; mindset, willing to make rapid, pragmatic decisions to improve performance, accelerate progress or magnify impact; recognizing that the &lsquo;good&rsquo; is not the enemy of the &lsquo;perfect&rsquo;</li>
<li><br></li>
<li>Direct, hands-on experience managing the process of building a model, incorporating features, tuning performance, and deploying into production</li>
<li><br></li>
<li>Knowing how and when to make trade-offs between incremental accuracy and exception-free performance</li>
<li><br></li>
<li>An ability to communicate the solution, its parameters, its assumptions and its inner workings to heterogeneous audiences, who vary widely in their levels of technical fluency and seniority</li>
<li><br></li>
<li>Comfort with working with distributed teams on code-based deliverables, using version control systems and code reviews</li>
<li><br></li>
</ul>
<br>
</li>
<li><br></li>
<li>Solid and battle-tested understanding of the standard canon of machine learning practices, including but not limited to:<br><ul>
<li><br></li>
<li>Linear and Logistic Regression</li>
<li><br></li>
<li>Decision Tree classification and regression</li>
<li><br></li>
<li>Neural Networks</li>
<li><br></li>
<li>Dimensionality Reduction</li>
<li><br></li>
<li>Clustering</li>
<li><br></li>
<li>Bagging, Boosting, and other non-parametric methods</li>
<li><br></li>
</ul>
<br>
</li>
<li><br></li>
<li>Demonstrated expertise working with and maintaining open source data analysis platforms, including but not limited to:<br><ul>
<li><br></li>
<li>Pandas, Scikit-Learn, Matplotlib, TensorFlow, Jupyter and other Python data tools</li>
<li><br></li>
<li>Spark, HDFS, Kafka and other high volume data tools</li>
<li><br></li>
<li>SQL and NoSQL storage tools, such as MySQL, Postgres, Cassandra, MongoDB and ElasticSearch</li>
<li><br></li>
</ul>
<br>
</li>
<li><br></li>
</ul><br><br>
via Remote Jobr Programming Jobs http://ift.tt/1WipnMk<br>
July 24, 2017 at 05:16PM
